### ISSUE TITLE
Align the GOVERNANCE.md and CONTRIBUTING.md documents

### ISSUE THREAD

**Issue by didier-wenzek** [thin-edge/thin-edge.io#1378]
The governance model has been approved with this follow-up task:

- move the repository rules from the governance.md to the contributing.md document.

_Originally posted by @TheNeikos in https://github.com/thin-edge/thin-edge.io/pull/1203#discussion_r955893433_

**Comment by matthiasbeyer** [comment]
I will do this in a PR after #1203 is merged.

**Comment by matthiasbeyer** [comment]
Was closed automatically with the merge of #1379, but as @didier-wenzek said we shouldn't close this one until the CONTRIBUTING.md was rewritten because of duplicated stuff in there now.
